tests/test_visitors/test_tokenize/test_primitives/test_numbers/test_number_float_zero.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from wemake_python_styleguide.violations.consistency import FloatZeroViolation
+from wemake_python_styleguide.visitors.tokenize.primitives import (
+    WrongNumberTokenVisitor,
+)
+
+
+def test_float_zero(
+    parse_tokens,
+    assert_errors,
+    default_options,
+    primitives_usages,
+    mode,
+):
+    """Ensures that float zeros (0.0) raise a warning."""
+    primitive = 0.0  # noqa: WPS358
```

Comment:
You can use just `file_tokens = parse_tokens(mode(primitives_usages.format('0.0')))`